src/components/missions/Mission.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-vars */
+import { useDispatch } from 'react-redux';
+
+const Mission = (mission) => {
+  const {
+    mission_id, mission_name, description,
+  } = mission;
+
+  const dispatch = useDispatch();
```

Comment:
* You could use destructuring like this in order to avoid disabling eslint rules
`const { mission_name: missionName, description }`
* Please consider removing code you will not need, so as to enforce **YAGNI** principles ( `useDispatch` and `mission_id`). It will also make your codebase cleaner and lighter.
* After doing these you should be able to remove the rules you have disabled on line one and two